### PR TITLE
ColorCommand is bacc

### PR DESCRIPTION
**Pull request by hechfx**

### CODE REVIEW

File: loritta-plugins/lori-guild-stuff/src/main/kotlin/net/perfectdreams/loritta/plugin/loriguildstuff/commands/ColorCommand.kt [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+                reply(
+                        LorittaReply(
+                                "Este comando é apenas para doadores, se você quer me ajudar a comprar um :custard:, então vire um doador! https://loritta.website/donate",
+                                "<:error:738489644039012506>"
+                        )
+                )
+            } else {
+                val selection = args.joinToString(" ")
+
+                if (colors[selection] !== null) {
```
Comment (MrPowerGamerBR) [inline loritta-plugins/lori-guild-stuff/src/main/kotlin/net/perfectdreams/loritta/plugin/loriguildstuff/commands/ColorCommand.kt:47]: Use `!=`

File: loritta-plugins/lori-guild-stuff/src/main/kotlin/net/perfectdreams/loritta/plugin/loriguildstuff/commands/ColorCommand.kt [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+                    val c = colors[selection]!!
+                    println(c)
```
Comment (MrPowerGamerBR) [inline loritta-plugins/lori-guild-stuff/src/main/kotlin/net/perfectdreams/loritta/plugin/loriguildstuff/commands/ColorCommand.kt:49]: owo debug

File: loritta-plugins/lori-guild-stuff/src/main/kotlin/net/perfectdreams/loritta/plugin/loriguildstuff/commands/ColorCommand.kt [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+                        LorittaReply(
+                                "Este comando é apenas para doadores, se você quer me ajudar a comprar um :custard:, então vire um doador! https://loritta.website/donate",
+                                "<:error:738489644039012506>"
+                        )
+                )
+            } else {
+                val selection = args.joinToString(" ")
+
+                if (colors[selection] !== null) {
+                    val c = colors[selection]!!
```
Comment (MrPowerGamerBR) [inline loritta-plugins/lori-guild-stuff/src/main/kotlin/net/perfectdreams/loritta/plugin/loriguildstuff/commands/ColorCommand.kt:48]: Would be better to `val c = colors[selection]` before the check and then do `c != null`

File: loritta-plugins/lori-guild-stuff/src/main/kotlin/net/perfectdreams/loritta/plugin/loriguildstuff/commands/ColorCommand.kt [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+                    println(c)
+                    val role = guild.getRoleById(c)!!
+
+                    if (member.roles.contains(role)) {
+                        guild.removeRoleFromMember(member, role).await()
+                        reply (
+                                LorittaReply(
+                                        "Cor removida!",
+                                        ":art:"
```
Comment (MrPowerGamerBR) [inline loritta-plugins/lori-guild-stuff/src/main/kotlin/net/perfectdreams/loritta/plugin/loriguildstuff/commands/ColorCommand.kt:57]: Replace with the unicode character (`\:art` on chat)

File: loritta-plugins/lori-guild-stuff/src/main/kotlin/net/perfectdreams/loritta/plugin/loriguildstuff/commands/ColorCommand.kt [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+                            list += ", ${color.key}"
+                        }
+                    }
+                }
+
+                if (args.isEmpty()) {
+                    reply (
+                            LorittaReply(
+                                    "Cores disponíveis: $list",
+                                    ":art:"
```
Comment (MrPowerGamerBR) [inline loritta-plugins/lori-guild-stuff/src/main/kotlin/net/perfectdreams/loritta/plugin/loriguildstuff/commands/ColorCommand.kt:87]: Replace with the unicode character (`\:art` on chat)

File: loritta-plugins/lori-guild-stuff/src/main/kotlin/net/perfectdreams/loritta/plugin/loriguildstuff/commands/ColorCommand.kt [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+                        reply (
+                                LorittaReply(
+                                        "Cor adicionada!",
+                                        ":art:"
+                                )
+                        )
+                    }
+                }
+
+                var list = ""
```
Comment (MrPowerGamerBR) [inline loritta-plugins/lori-guild-stuff/src/main/kotlin/net/perfectdreams/loritta/plugin/loriguildstuff/commands/ColorCommand.kt:71]: var list = colors.joinToString(", ")

File: loritta-plugins/lori-guild-stuff/src/main/kotlin/net/perfectdreams/loritta/plugin/loriguildstuff/commands/ColorCommand.kt [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+
+                val c = colors[selection]
+
+                if (c != null) {
+                    val role = guild.getRoleById(c)!!
+
+                    if (member.roles.contains(role)) {
+                        guild.removeRoleFromMember(member, role).await()
+                        reply (
+                                LorittaReply(
```
Comment (MrPowerGamerBR) [inline loritta-plugins/lori-guild-stuff/src/main/kotlin/net/perfectdreams/loritta/plugin/loriguildstuff/commands/ColorCommand.kt:54]: Remove the white space between the `reply` and `(`

File: loritta-plugins/lori-guild-stuff/src/main/kotlin/net/perfectdreams/loritta/plugin/loriguildstuff/commands/ColorCommand.kt [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+                                        "Cor removida!",
+                                        "\uD83C\uDFA8"
+                                )
+                        )
+                    } else {
+                        guild.addRoleToMember(member, role).await()
+                        reply (
```
Comment (MrPowerGamerBR) [inline loritta-plugins/lori-guild-stuff/src/main/kotlin/net/perfectdreams/loritta/plugin/loriguildstuff/commands/ColorCommand.kt:61]: Same as above

File: loritta-plugins/lori-guild-stuff/src/main/kotlin/net/perfectdreams/loritta/plugin/loriguildstuff/commands/ColorCommand.kt [hunk 9]
```diff
@@ -0,0 +1,83 @@
...
+                                        "\uD83C\uDFA8"
+                                )
+                        )
+                    }
+                }
+
+                val list = colors.keys.joinToString(", ")
+
+                if (args.isEmpty()) {
+                    reply (
```
Comment (MrPowerGamerBR) [inline loritta-plugins/lori-guild-stuff/src/main/kotlin/net/perfectdreams/loritta/plugin/loriguildstuff/commands/ColorCommand.kt:73]: Same as above

File: loritta-plugins/lori-guild-stuff/src/main/kotlin/net/perfectdreams/loritta/plugin/loriguildstuff/commands/ColorCommand.kt [hunk 10]
```diff
@@ -0,0 +1,83 @@
...
+    )
+
+    fun create(loritta: LorittaDiscord) = discordCommand(loritta, listOf("cor", "color"), CommandCategory.MISC) {
+        this.hideInHelp = true
+        this.commandCheckFilter { lorittaMessageEvent, _, _, _, _ ->
+            lorittaMessageEvent.guild?.idLong == 297732013006389252L
+        }
+
+        executesDiscord {
+
```
Comment (MrPowerGamerBR) [inline loritta-plugins/lori-guild-stuff/src/main/kotlin/net/perfectdreams/loritta/plugin/loriguildstuff/commands/ColorCommand.kt:32]: Remove empty whitespace

File: loritta-plugins/lori-guild-stuff/src/main/kotlin/net/perfectdreams/loritta/plugin/loriguildstuff/LoriGuildStuffPlugin.kt [hunk 11]
```diff
@@ -15,9 +16,7 @@ class LoriGuildStuffPlugin(name: String, loritta: LorittaBot) : LorittaDiscordPl
 
     override fun onEnable() {
         loritta as LorittaDiscord
-        registerCommand(NotifyBackgroundsCommand.create(loritta))
-        registerCommand(NotifyCommand.create(loritta))
-        registerCommand(AddBackgroundCommand.create(loritta))
+        registerCommands(NotifyBackgroundsCommand.create(loritta), NotifyCommand.create(loritta), AddBackgroundCommand.create(loritta), ColorCommand.create(loritta))
```
Comment (MrPowerGamerBR) [inline loritta-plugins/lori-guild-stuff/src/main/kotlin/net/perfectdreams/loritta/plugin/loriguildstuff/LoriGuildStuffPlugin.kt:19]: Please put all commands in different lines
```kotlin
registerCommands(
    NotifyBackgroundsCommand.create(loritta),
    ...
    ColorCommand.create(loritta)
)
```

File: loritta-plugins/lori-guild-stuff/src/main/kotlin/net/perfectdreams/loritta/plugin/loriguildstuff/commands/ColorCommand.kt [hunk 12]
```diff
@@ -0,0 +1,83 @@
...
+            val donatorRole = guild.getRoleById(364201981016801281L)!! //364201981016801281L - role original
+            val member = this.member!!
+
+            if (!member.roles.contains(donatorRole)) {
+                reply(
+                        LorittaReply(
+                                "Este comando é apenas para doadores, se você quer me ajudar a comprar um :custard:, então vire um doador! https://loritta.website/donate",
+                                "<:error:412585701054611458>"
```
Comment (Nightdavisao) [inline loritta-plugins/lori-guild-stuff/src/main/kotlin/net/perfectdreams/loritta/plugin/loriguildstuff/commands/ColorCommand.kt:40]: Use [Constants](https://github.com/LorittaBot/Loritta/blob/development/loritta-discord/src/main/java/com/mrpowergamerbr/loritta/utils/Constants.kt).ERROR instead

**Comment by MrPowerGamerBR** [comment]
LGTM

**Pull request merged**